app/app.js

```diff
@@ -61,8 +61,20 @@ function load(stats) {
 			if(!m) return;
 			origin.moduleUid = m.uid;
 		});
+		chunk.modules.forEach(function(module) {
+			var m = mapModulesIdent["$"+module.identifier], s;
+			if(!m) return;
+			s = m.dependencies.reduce(function(totalSize, dep) {
```

Comment:
What if there's a dependency shared between multiple modules in the chunk? It seems like there's potential to over-count in that case. Could be remedied by tracking whether a given module was already incorporated into the total size calculation.